### ISSUE TITLE
[Website]: SVG image usage

### ISSUE THREAD

**Issue by joshbruce** [BrianGilmanGSA/sam-web-design-standards-archive#55]
The copy around imagery:

http://briangilmangsa.github.io/sam-web-design-standards/visual-style/

There is mention of using SVG graphics whenever possible. This recommendation was implemented in one of the application; however, the implementation is not accessible using neither Section 508 nor WCAG 2.0. Expand language to describe use in more detail.
- [ ] Create SVG graphic.
- [ ] Create PNG or JPEG alternative.
- [ ] If embedded directly (not through css), use ARIA`labels` or `describedby` functionality to describe image content of SVG, use `alt` attribute on PNG image. (Lean toward PNG or JPEG for embedded images.)
- [ ] Confirm compliance for other images - create new issue if other site imagery does not comply.
